security/integration/main.go

```diff
@@ -143,43 +143,41 @@ func initializeIntegrationTest(cmd *cobra.Command, args []string) error {
 
 func runSelfSignedCATests() error {
 	// Create Istio CA with self signed certificates
-	self_signed_ca_pod, err := utils.CreatePod(opts.clientset, opts.namespace,
+	selfSignedCaPod, err := utils.CreatePod(opts.clientset, opts.namespace,
 		fmt.Sprintf("%v/istio-ca:%v", opts.containerHub, opts.containerTag), "istio-ca-self")
 	if err != nil {
 		return fmt.Errorf("failed to deploy Istio CA (error: %v)", err)
 	}
-	glog.Infof("Succesfully created Istio CA pod(%v) with the self-signed-ca option",
-		self_signed_ca_pod.GetName())
+	glog.Infof("Successfully created Istio CA pod(%v) with the self-signed-ca option",
+		selfSignedCaPod.GetName())
 
 	// Test the existence of istio.default secret.
-	if s, err := utils.WaitForSecretExist(opts.clientset, opts.namespace, "istio.default",
-		secretWaitTime); err != nil {
-		return err
-	} else {
-		glog.Info(`Secret "istio.default" is correctly created`)
-
-		expectedID := fmt.Sprintf("spiffe://cluster.local/ns/%s/sa/default", s.GetNamespace())
-		examineSecret(s, expectedID)
+	secret, error := utils.WaitForSecretExist(opts.clientset, opts.namespace, "istio.default",
```

Comment:
I think 'err' is widely used since 'error' can be confused with some lib name. At least please be consistent within this file (there are some 'err' and some 'error').

Reply:
I replaced err with errors in this file.